.gitlab/custom-jobs-and-variables.yml

```diff
@@ -31,7 +31,7 @@ variables:
 # Project specific variants for poodle
   PROJECT_POODLE_VARIANTS: "~shared +openmp +vectorization +tests"
 # Project specific deps for poodle
-  PROJECT_POODLE_DEPS: ""
+  PROJECT_POODLE_DEPS: "^blt@develop "
```

Comment:
@long58 @rhornung67 let’s not deal with this in this PR, but blt@develop is heavily used in the CI. We should check whether that is still necessary, it’s not good to point at a moving target in CI.